src/app/views/job-details/job-details.component.ts

```diff
@@ -1,35 +1,109 @@
-import {Component} from '@angular/core';
+import {Component, OnInit, Input} from '@angular/core';
 import {ActivatedRoute} from '@angular/router';
 import {Job} from '../../models/job/job';
 import {JobProxy} from '../../services/proxy/job-proxy.service';
 import {UserManager} from '../../services/user-manager.service';
 import {User} from '../../models/user';
 import {TranslationService} from '../../services/translation.service';
 import {TranslationListener} from '../../components/translation.component';
+import {UserProxy} from '../../services/proxy/user-proxy.service';
+import {UserJob} from '../../models/user/user-job';
+import {Router} from '@angular/router';
+import {UserBankAccount} from '../../models/user/user-bank-account';
 
 @Component({
   templateUrl: './job-details.component.html',
   styleUrls: ['./job-details.component.scss'],
   providers: [JobProxy]
 })
-export class JobDetailsComponent extends TranslationListener {
+export class JobDetailsComponent extends TranslationListener implements OnInit {
+  @Input() isInUserView: boolean;
+  @Input() userJobId: number;
   private job: Job;
   private currentJobId: number;
   user: User;
   isCompanyUser: boolean;
+  userJob: UserJob;
+  countOfApplicants: number = 0;
+  showCandidateBankAccountDetails: boolean = false;
+  showTermsAccept: boolean = false;
+  acceptTerms: boolean = false;
+  bankAccount: UserBankAccount = new UserBankAccount();
+  errors: any = {};
+  jobDetailsVisible: boolean;
 
-  constructor(private route: ActivatedRoute, private jobProxy: JobProxy, private userManager: UserManager, protected translationService: TranslationService) {
+  constructor(private route: ActivatedRoute, private jobProxy: JobProxy, private userManager: UserManager, private userProxy: UserProxy, private router: Router, protected translationService: TranslationService) {
     super(translationService);
     this.isCompanyUser = this.userManager.isCompanyUser();
+    this.user = userManager.getUser();
     this.route.params.subscribe(params => {
       this.currentJobId = parseInt(params['id']);
-      this.loadData();
     });
+    this.jobDetailsVisible = true;
+  }
+
+  ngOnInit() {
+    this.loadData();
   }
 
   loadData() {
     this.jobProxy.getJob(this.currentJobId, {include: 'owner,company,hourly_pay,company.company_images,comments'}).then(result => {
       this.job = result;
+      this.getJobInfo();
     });
   }
+
+  getJobInfo() {
+    if (this.isCompanyUser && this.job.company.id === this.user.company.id) {
+      this.jobProxy.getJobUsers(this.job.id, { include: 'user', 'filter[accepted]': true }).then(response => {
+        this.userJob = response[0];
+        console.log(response);
```

Comment:
Does the knife mean "cut"? :) if so I get it, remove the console output. Missed that, didn't change the code in those files since I just copied the content from job-status to job-details to merge them. I didn't think to fix his code, I'll take a look at it!